config/samples/intel_v1alpha1_ovms.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   models_settings:
     single_model_mode: true
     config_configmap_name: ""
-    model_config: ""   
+    model_config_path: ""
```

Comment:
I think it's just `config_path`:
https://github.com/openvinotoolkit/operator/blob/main/bundle/manifests/openvino-operator.clusterserviceversion.yaml#L109-L115

Reply:
same here:
https://github.com/openvinotoolkit/operator/blob/main/helm-charts/ovms/values.yaml#L28